src/test/java/tests/Task40.java

```diff
@@ -0,0 +1,185 @@
+package tests;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.github.bonigarcia.wdm.config.DriverManagerType;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Task40 {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @BeforeEach
+    void start() {
+        WebDriverManager.getInstance(DriverManagerType.CHROME).setup();
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().timeouts().pageLoadTimeout(Duration.ofSeconds(10));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+
+    @ParameterizedTest
+    @CsvSource({"siarheikost , 1qaz2wsx!QAZ@WSX", "seregeikss, zaq1xsw2ZAQ!XSW@"})
+    void login(String name, String psw) throws InterruptedException {
+        driver.get("https://passport.yandex.com/auth/add?retpath=https%3A%2F%2Fpassport.yandex.com%2Fprofile&noreturn=1");
+
+        driver.findElement(By.id("passp-field-login")).sendKeys(name);
+        driver.findElement(By.cssSelector(".Button2.Button2_size_l.Button2_view_action")).click();
+        Thread.sleep(1500);//(ExplicitlyWait)
+        driver.findElement(By.xpath("//input[@dir = 'ltr']")).sendKeys(psw);
+        driver.findElement(By.cssSelector(".Button2.Button2_size_l.Button2_view_action")).click();
+        Thread.sleep(2500);//(ExplicitlyWait)
+
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".personal-info-login__text")));
+    }
+
+    @Test
+    void alertBoxTest() {
+        driver.get("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+        driver.findElement(By.xpath("//button[@onclick  ='myAlertFunction()']")).click();
+        Alert alert = driver.switchTo().alert();
```

Comment:
please think about how not to duplicate code across your tests, especially according alert.